synthetic_data_experiments/full_county_configs/full_county_bottom_heavy_0point25.ini

```
@@ -0,0 +1,132 @@
+# The configuration file for DAS as run in standalone mode by das_decennial/run_1940_standalone.sh
+# This is a modified copy of das_decennial/E2E_1940_CONFIG.sh
+
+[DEFAULT]
+# root specifies the root location for all files
+# For the demo, the root in the current directory
+name: 2018DAS_E2E_1940
+root: .
+loglevel: INFO
+logfolder: logs
+
+
+[ENVIRONMENT]
+DAS_FRAMEWORK_VERSION: 0.0.1
+GRB_ISV_NAME: Census
+GRB_APP_NAME: DAS
+GRB_Env3: 0
+GRB_Env4:
+
+[geodict]:
+# Names of smallest to largest geocode (no spaces)
+geolevel_names: Enumdist,County,State,National
+# Largest geocode length to smallest, put 1 for top level (no spaces)
+geolevel_leng: 10,6,2,1
+
+[setup]
+setup: programs.das_setup.setup
+
+# Spark config stuff
+spark.name: DAS_E2E
+#local[6] tells spark to run locally with 6 threads
+#spark.master: local[9]
+#Error , only writes to log if there is an error (INFO, DEBUG, ERROR)
+spark.loglevel: ERROR
+
+[reader]
+ipums_file: $EXT1940USCB
+# package(s).module_name.class_name of the reader module
+reader: programs.reader.e2e_1940_reader.reader
+
+###
+### List of tables
+### These tables have decennial Census specific process methods
+### Table class methods will likely need to be rewritten for other applications
+### 
+tables: PersonData UnitData
+
+privacy_table: PersonData
+constraint_tables: UnitData
+
+# table_name.path - location of dir of filename=======
+
+PersonData.geography: geocode
+PersonData.histogram: hhgq age18plus hispanic race
+
+UnitData.geography: geocode
+UnitData.histogram: hhgq
+
+[engine]
+engine: programs.engine.topdown_engine.engine
+
+# should we delete the true data after making DP measurments (1 for True or 0 for False)
+delete_raw: 1
+
+[budget]
+epsilon_budget_total: 10.0
+global_sensitivity: 2.0
+
+#budget in topdown order (e.g. County, Tract, Block Group, Block)
+geolevel_budget_prop: 0.00625,0.00625,0.0125,0.975
```

Shouldn't this be reversed?
```suggestion
geolevel_budget_prop: 0.0125,0.00625,0.00625,0.975
```